docs/src/content/docs/getting-started/usage.mdx

```diff
@@ -0,0 +1,184 @@
+	 */
+	performCount: number;
+};
+```
+
+## `TaskInstance` structure
+
+- `error`: if an error is thrown inside the task instance, it will be found here
+- `isCanceled`: whether the task instance was canceled
+- `isError`: whether the task instance throw an error before completing
+- `isRunning`: whether the task instance is currently running
+- `isSuccessful`: whether the task instance completed successfully
+- `value`: if the task instance completed successfully, this will be the return value
+
+```ts
+type TaskInstance<TReturn> = {
+	/**
+	 * If an error is thrown inside the task instance, it will be found here.
+	 */
+	error?: Error;
+
+	/**
+	 * Indicates whether the task instance was canceled.
+	 */
+	isCanceled: boolean;
+
+	/**
+	 * Indicates whether the task instance threw an error before completing.
+	 */
+	isError: boolean;
+
+	/**
+	 * Indicates whether the task instance is currently running.
+	 */
+	isRunning: boolean;
+
+	/**
+	 * Indicates whether the task instance completed successfully.
+	 */
+	isSuccessful: boolean;
+
+	/**
+	 * If the task instance completed successfully, this will be the return value.
+	 */
+	value?: TReturn;
+};
+```
+
+## Task modifiers
+
+`@sheepdog/svelte` allows you to specify a task modifier that changes the behavior of the perform
+function. By default when you call `perform` the task will be executed immediately regardless if
+there are other instances of the same task in execution. For example, you could use the `enqueue`
+modifier to instruct `@sheepdog/svelte` to execute the task instances one after another (in a
+queue - who would've thought)
+
+Here's how you can specify the modifier:
+
+<Tabs syncKey="examples">
+	<TabItem label="Dot notation">
+	```svelte ins=".enqueue" {4}
+	<script>
+		import { task, timeout } from '@sheepdog/svelte';
+
+    	const myTask = task.enqueue(async () => {
+    		// async stuff
+    	});
+    </script>
+
+    <button on:click={()=>{
+    	myTask.perform();
+    }}>
+    	do async stuff one by one
+    </button>
+    ```
+    </TabItem>
+    <TabItem label="Option notation">
+    ```svelte ins="enqueue" {6}
+    <script>
+    	import { task, timeout } from '@sheepdog/svelte';
+
+    	const myTask = task(async () => {
+    		// async stuff
+    	}, { kind: "enqueue" });
+    </script>
+
+    <button on:click={()=>{
+    	myTask.perform();
+    }}>
+    	do async stuff one by one
+    </button>
+    ```
+    </TabItem>
+
+</Tabs>
+
+<LinkCard
+	href="/guides/task-modifiers/"
+	title="Task modifiers"
+	description="Read everything about task modifiers in our guide"
+/>
+
+There are currently five task modifiers that you can apply to your task.
+
+- `default`: the default behavior, every task will run immediately
+- `drop`: if other tasks are already running the new instance will be immediately canceled
+- `enqueue`: every task instance is stored in a queue and executed with a
+  [FIFO](<https://en.wikipedia.org/wiki/FIFO_(computing_and_electronics)>) strategy
```

Comment:
Do you know if it's possible to get this to open in a new window?

Reply:
Good idea...will fix